### PR TITLE
fix: handle error when migrating automation action

### PR DESCRIPTION
**Pull request by rileymd88**
This fixes an issue where the automation action does not have an `automationId` property set up yet which can cause the automation action to fail

### CODE REVIEW

File: src/utils/automation-props.js [hunk 1]
```diff
@@ -44,7 +44,8 @@ const getAutomationProps = (multiUserAutomation, getAutomations) => ({
     translation: "Object.ActionButton.Automation.Link",
     component: "link",
     url: (data) => `/automations/editor/${data.automationId}`,
-    show: (data) => data.automationId.length > 1 && multiUserAutomation,
+    show: (data) =>
+      Object.prototype.hasOwnProperty.call(data, "automationId") && data.automationId.length > 1 && multiUserAutomation,
```
Comment (niekvanstaveren) [inline src/utils/automation-props.js:48]: Can't you just check existence of automationId with ?

**Review by niekvanstaveren — COMMENTED** [review]
Is this still an issue?

**Comment by rileymd88** [comment]
> Is this still an issue?

Yes

File: src/utils/automation-props.js (hunk 1 above)
Comment (rileymd88) [reply src/utils/automation-props.js:48]: I will update this

**Review by niekvanstaveren — APPROVED** [review]

**Pull request closed without merging**